src/common/services/translator.ts

```diff
@@ -28,8 +28,8 @@ const initI18n = i18n.init({
 
 export enum AvailableLanguages {
     en = "English",
-    fr = "Français",
-    de = "Deutch",
+    fr = "French",
```

Comment:
What is the rationale for changing from French to English here?

Reply:
the reason is to standardize languages in english which is the default language.

Reply:
i made some changes in translation files in order to translate each languages when it is selected in settings